packages/next-app/src/contexts/AdminToolsContext.tsx

```diff
@@ -17,6 +18,17 @@ export function toSlug(string?: string) {
   );
 }
 
+export function toCammelCase(string: string) {
```

Comment:
```suggestion
export function toCamelCase(string: string) {
```

Reply:
Also this is the sort of thing that's so abstract it should be abstracted away from the component code to avoid other folks reimplementing the same thing.